agent/task.go

```diff
@@ -146,26 +149,31 @@ func (a *Agent) runPreTaskCommands(ctx context.Context, tc *taskContext) {
 		taskGroup, err := model.GetTaskGroup(tc.taskGroup, tc.taskConfig)
 		if err != nil {
 			tc.logger.Execution().Error(errors.Wrap(err, "error fetching task group for pre-group commands"))
-			return
+			return nil
 		}
 		if taskGroup.SetupGroup != nil {
-			err = a.runCommands(ctx, tc, taskGroup.SetupGroup.List(), false)
+			err = a.runCommands(ctx, tc, taskGroup.SetupGroup.List(), taskGroup.SetupGroupFailTask)
 			if err != nil {
-				tc.logger.Execution().Error(errors.Wrap(err, "error running task setup group"))
+				err = errors.Wrap(err, "error running task setup group")
+				tc.logger.Execution().Error(err)
```

Comment:
This error goes to the task logs, so don't return a wrapped error. (People don't want internal Evergreen stacks in their tasks.)

Reply:
I'm assuming you mean to still log the wrapped error like before, but return the unwrapped error? Let me know if I'm misunderstanding.